src/main/java/racingcar/manager/UIManager.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.manager;
+
+import racingcar.ui.UIInGame;
+import racingcar.ui.UIIntro;
+import racingcar.util.Input;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UIManager {
+    UIIntro intro = new UIIntro();
+    UIInGame inGame = new UIInGame();
+    Input input = new Input();
+    SecurityManager securityManager = new SecurityManager();
+    public int showIntro(){
+        intro.printIntro();
+        return checkValidIntroInputNum();
+    }
+    private int checkValidIntroInputNum() {
```

Comment:
```suggestion
    }
    
    private int checkValidIntroInputNum() {
```

빈 공백 라인을 추가하는 것이 관례임